clicache/src/DataInput.cpp

```diff
@@ -96,7 +98,7 @@ namespace Apache
           _GEODE_NEW(m_buffer, System::Byte[len]);
           pin_ptr<const Byte> pin_buffer = &buffer[0];
           memcpy(m_buffer, (void*)pin_buffer, len);
-          m_nativeptr = gcnew native_conditional_unique_ptr<native::DataInput>(m_cache->GetNative()->createDataInput(m_buffer, len));
+          m_nativeptr = gcnew native_conditional_unique_ptr<native::DataInput>(std::unique_ptr<native::DataInput>(new native::DataInput(m_cache->GetNative()->createDataInput(m_buffer, len))));
```

Comment:
Would have expected this to have been "formatted" as well.  Any reason why the suggestion to use `std::make_unique` was not used?

Reply:
Nope, just missed the note about being safe to use that in CLI code.  Will fix.